maths/simpson_rule.py

```diff
@@ -13,46 +13,54 @@ def method_2(boundary: list[int], steps: int) -> float:
     # "Simpson Rule"
     # int(f) = delta_x/2 * (b-a)/3*(f1 + 4f2 + 2f_3 + ... + fn)
     """
-    Calculate the definite integral of a function using Simpson's Rule.
+        Calculate the definite integral of a function using Simpson's Rule.
```

Comment:
Please revert all the extra indentation.  It can be difficult to write complex tests in 80 characters and it is even more difficult with just 76.